src/components/List/README.md

```diff
@@ -4,30 +4,30 @@
+| sortable                                   | `Boolean`         |          |           | Флаг, включающий сортировку списка.                                                                                                                                                                        |
+| sortHandleAlign                            | `left` `right`    |          |           | Выравнивание индикатора сортировки (слева или справа).                                                                                                                                                     |
+| onSortEnd                                  | `Function`        |          |           | Обработчик события сортировки. `({oldIndex: number, newIndex: number}) => void`                                                                                                                            |
+| virtualized                                | `Boolean`         |          |true       | Флаг, включающий виртуализацию. При выключенном флаге будут отрисованы все элементы разом.                                                                                                                 |
+| onItemClick                                | `Function`        |          |           | Обработчик клика по элементу. `(item: any, index: number, fromKeyboard?: bool) => void`                                                                                                                    |
+| deactivateOnLeave                          | `Boolean`         |          |true       | При выставленном флаге выделение элемента пропадает при уходе курсора с элемента или потере списком фокуса, при снятом - последний выделенный элемент остается выделенным всегда.                          |
+| activeItemIndex                            | `Number`          |          |           | При заданном значении элемент с этим индексом рендерится как активный ~~пока проклятие не будет снято~~.                                                                                                   |
+| selectedItemIndex                          | `Number`          |          |           | При заданном значении элемент с этим индексом рендерится как выбранный (цвет фона из `--yc-color-selection`).                                                                                              |
+| itemClassName                              | `String`          |          |           | Кастомное имя класса которое будет добавлено к контейнеру элемента                                                                                                                                         |
+| itemsClassName                             | `String/Function` |          |           | Кастомное имя класса которое будет добавлено к списку элементов (или функция, возвращающая имя класса)                                                                                                     |
```

Comment:
I guess it should be translated to english

Reply:
U can help with translation